sql/6_1_0-to-7_0_0_upgrade.sql

```diff
@@ -656,3 +656,13 @@ DROP TABLE `temp_table_one`;
 #IfNotRow2D list_options list_id issue_subtypes option_id diagnosis
 INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`) VALUES ('issue_subtypes','diagnosis','Diagnosis',40);
```

Comment:
this is just a draft, so figured you haven't added to database.sql yet

Reply:
I should now have all the changes in the database.sql file.